exercises/practice/hamming/test/hamming_test.dart

```diff
@@ -1,17 +1,12 @@
 import 'package:hamming/hamming.dart';
 import 'package:test/test.dart';
 
-final hamming = Hamming();
-
-/// Test cases match one-to-one with those described
-/// in the problem specification found here:
-/// https://github.com/exercism/problem-specifications/blob/master/exercises/hamming/canonical-data.json
 void main() {
+  final hamming = Hamming();
+
   final mustEqual = predicate(
       (Error e) => e is ArgumentError && e.message == 'left and right strands must be of equal length',
       'an ArgumentError with the message "left and right strands must be of equal length"');
```

Comment:
Based on @devkabiir's other post, the strings will need to be update to reference "first" and "second" instead of left and right.

Reply:
I updated to match what was in problem-specifications, which removes left/right altogether.